requirements.txt

```diff
@@ -6,6 +6,7 @@ jinja2
 keystoneauth1
 python-keystoneclient
 prometheus-client==0.0.13
+regex
```

Comment:
`re` is a standard Python library, so you don't need to include any additional package requirements like this.